src/theme/variables.scss

```diff
@@ -19,6 +19,9 @@ $app-direction: ltr;
 // To view all the possible Ionic variables, see:
 // http://ionicframework.com/docs/theming/overriding-ionic-variables/
 
+$background-color: #ecf4f6;
```

Comment:
Could we add these colours to the `$colors` map so we have a defined list of used colours in the app?

Reply:
It might be good idea to separate different categories into different files and just import it here at `variables.scss`